### DISCUSSION TITLE
Add Mastodon website verification setting

### DISCUSSION THREAD

**Discussion by KHannibal** [Lednerb/bilberry-hugo-theme#440] [Ideas]
Hi,

Mastodon allows users to verify a website that is linked in their profile in order to show that it is owned by them. This can be done by adding a link on their website that contains a `rel="me"` attribute.

More info in Mastodon's documentation (bottom of the page) : https://docs.joinmastodon.org/user/profile/#fields

It would be nice if Bilberry allowed users to verify their site in a convenient and trivial way. Thank you !

**Comment by igor-baiborodine** [comment]
@KHannibal, moving to [Discussions/Ideas](https://github.com/Lednerb/bilberry-hugo-theme/discussions/categories/ideas).

**Comment by JoeCotellese** [comment]
Just came here looking for this too.

**Comment by igor-baiborodine** [comment]
@KHannibal, @JoeCotellese Implemented with PR https://github.com/Lednerb/bilberry-hugo-theme/pull/445.

Usage: `link = "https://twitter.com/TheRealLednerb", icon = "fab fa-twitter", rel="me"`

**Reply by KHannibal** [reply]
Cool ! I'll try this out ! Thanks @JoeCotellese @igor-baiborodine !

**Comment by KHannibal** [comment]
Works great, thank you !
